extensions/BMO/template/en/default/hook/bug_modal/edit-custom_field-cf_crash_signature.html.tmpl

```diff
@@ -28,7 +28,10 @@
           sig FILTER html;
         END;
         "</div>";
-      END;
+      END %]
+      [%# socorro_lens_url() is defined in Bugzilla/Template.pm %]
+      <iframe src='[% socorro_lens_url(split_cf_crash_signature) FILTER html %]' style='border:0; width:300px; height:200px; max-width:600px; max-height:300px; float:left;' id='chart' scrolling='no'></iframe>
```

Comment:
This element has both id='chart' *and* inline styles, rather than id='chart' and styles in a .css file, which seems odd in a non-BMO general context to me.